### PR TITLE
Added methods, changed endpoints, and separated into multiple cotrollers

### PR DESCRIPTION
**Pull request by jboud17**
#### Reviewers:
@josephjustn @jenfox @khsieh
#### Observers:
@jmknighten9 
#### Repository
hydra-batch-serviceBAM
#### Related Pull Requests:
none
#### Description:
- End points changed
- Methods added
- Controllers added
- End points added
#### Effect on Project or other Services:
Need to change Angular request endpoints

### CODE REVIEW

File: batch-service/src/main/java/com/revature/batch/controller/BatchController.java [hunk 1]
```diff
@@ -1,214 +1,54 @@
...
+/**
+ * 
+ * @author Josh Boudreau, Sonam Sherpa, Marko Miocic
+ * Batch: 1802-Feb12-java-matt
+ * 
+ * handles ZUUL endpoint /batches
+ *
+ */
 @RestController
+@RequestMapping(value = "/batch")
```
Comment (khsieh) [inline batch-service/src/main/java/com/revature/batch/controller/BatchController.java]: so full url for your controller from gateway to this controller is gonna be "localhost:9001/api/v2/batches/batch"? Is this what you want it to be? - OctoDad

File: batch-service/src/main/java/com/revature/batch/controller/BatchCrudController.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+/**
+ * 
+ * @author Josh Boudreau, Sonam Sherpa, Marko Miocic
+ * Batch: 1802-Feb12-java-matt
+ * 
+ * handles ZUUL endpoint /batches
+ *
+ */
+@RestController
```
Comment (khsieh) [inline batch-service/src/main/java/com/revature/batch/controller/BatchCrudController.java]: Missing a (value="") mapping here?

File: batch-service/src/main/java/com/revature/batch/controller/BatchController.java (hunk 1 above)
Comment (jboud17) [reply batch-service/src/main/java/com/revature/batch/controller/BatchController.java]: Yes, because all appending values will relate to a single batch

Comment (khsieh) [reply batch-service/src/main/java/com/revature/batch/controller/BatchController.java]: sweet, thanks for clarifying. 🍬

File: batch-service/src/main/java/com/revature/batch/controller/BatchCrudController.java (hunk 2 above)
Comment (jboud17) [reply batch-service/src/main/java/com/revature/batch/controller/BatchCrudController.java]: Does not need a value if it relates to /batches right?

Comment (khsieh) [reply batch-service/src/main/java/com/revature/batch/controller/BatchCrudController.java]: awesome! putting it through now! 🦈

**Pull request merged**